lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -289,6 +289,26 @@ class DecomposeAtenExpandOp : public OpRewritePattern<AtenExpandOp> {
 };
 } // namespace
 
+namespace {
+template<typename OpTy, typename T1T2Op>
+class DecomposeAtenAddCLikeOp : public OpRewritePattern<OpTy> {
+  using OpRewritePattern<OpTy>::OpRewritePattern;
+  LogicalResult matchAndRewrite(OpTy op,
+                                PatternRewriter &rewriter) const override {
+    Location loc = op.getLoc();
+    Value t = op.self();
```

Comment:
Instead of `t`, `t1` and `t2`, could you name it differently? Something like lhs, rhs and alpha which is more meaningful. Thanks.

Reply:
Hi, Sorry I didn't see this request.  I used these variable names to match the addcmul/addcdiv documentation.  I can definitely change them, but I don't know if lhs and rhs are great names as its elementwise multiplication and not matrix multiplication

Reply:
We can probably just use the names in the registry dump like `self/input` `tensor1` `tensor2` and add the formula in the comment `out=input​+value×tensor1​×tensor2​` so that people less familiar with the op can follow easily.